library/src/main/jni/Android.mk

```diff
@@ -22,5 +22,6 @@ LOCAL_MODULE    := glues
 LOCAL_SRC_FILES := com_panoramagl_opengl_GLUES.c
 LOCAL_LDLIBS    := -llog -lGLESv1_CM
 LOCAL_CFLAGS	:= -fopenmp
+LOCAL_LDFLAGS   += -Wl,-z,max-page-size=16384
```

Comment:
When I see https://youtu.be/MnMGJhuChRI?si=12jSCTjFTn98o4K9&t=212 and we are already on NDK 28+ I guess this is pointless @devsondroid